src/services/time-entries-api.js

```diff
@@ -1,25 +1,26 @@
+const apiUrl = `${
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:3000'
+    : 'https://my-json-server.typicode.com/humanoidsbv/team-awesome-dylan-api'
+}/time-entries`;
+
 export async function fetchTimeEntries() {
-  const response = await fetch(
-    'http://localhost:3000/time-entries?_sort=startTimestamp&_order=desc'
```

Comment:
Now you've removed the sorting, is this intentional?

Reply:
This was accidentally removed, reverted!